packages/web/src/components/nav/desktop/NavAudio.tsx

```diff
@@ -103,9 +91,10 @@ const NavAudio = () => {
             item !== 'none' && (
```

Comment:
not your code and not your problem but figured i'd point out for learning purposes: we're trying stick to our styleguide and avoid using `&&` in favor of ternaries.

Reply:
That's what I get for not reading the style guide yet.
I've typically been in favor of the `condition ? (component) : null` pattern. But I also did recently read something about that being an anti-pattern now due to the way React handles `null` vs `false`

Reply:
Oh wut. Please do share if you find that.

Reply:
Alright I did an overkill modification for you and just moved the logic to another component. It was kind of driving me nuts anyway.
https://github.com/AudiusProject/audius-client/pull/3218/commits/cfbb6283d8da8e4b709e2f3174a0b0662fed2403